core/core-dotspacemacs.el

```diff
@@ -386,7 +386,7 @@ List sizes may be nil, in which case
 (defvar dotspacemacs-frozen-packages '()
   "A list of packages that cannot be updated.")
 
-(defvar dotspacemacs-pretty-docs t
+(defvar dotspacemacs-pretty-docs nil
```

Comment:
Made it off by default to make it less surprising for people with legacy .spacemacs files.